index.bs

```diff
@@ -6268,11 +6256,11 @@ to its caller, and so on.
         Consider the following algorithm:
 
         1.  Let |x| be the ECMAScript value passed in to this algorithm.
-        1.  Let |y| be the result of calling [=ToString=](|x|).
+        1.  Let |y| be the result of calling <a abstract-op>ToString</a>(|x|).
```

Comment:
This is unfortunately necessary because of https://github.com/tabatkins/bikeshed/issues/809.